### PR TITLE
FIX: 354 setup

### PR DESCRIPTION
**Pull request by gkorompi**

### CODE REVIEW

File: tests/legacy/system/test_edb.py [hunk 1]
```diff
@@ -849,6 +849,8 @@ def test_hfss_simulation_setup(self):
         self.local_scratch.copyfolder(source_path, target_path)
         edbapp = Edb(target_path, edbversion=desktop_version)
         setup1 = edbapp.create_hfss_setup("setup1")
+        setup2 = edbapp.create_hfss_setup("setup1")
+        assert setup2 == None
```
Comment (SMoraisAnsys) [inline tests/legacy/system/test_edb.py:853]:
```suggestion
        assert not edbapp.create_hfss_setup("setup1")
```

[the remark this answers is not in the document]
Comment (SMoraisAnsys) [reply tests/legacy/system/test_edb.py:853]: I do agree with your point (and propose another change though)

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
Please fix the return value as pointed by @ring630 and either pick his proposition or mine for the test

**Review by ring630 — APPROVED** [review]
LGTM

**Review by SMoraisAnsys — APPROVED** [review]

**Pull request merged**